src/main/java/net/fabricmc/loader/metadata/V1ModMetadataParser.java

```diff
@@ -444,6 +444,9 @@ private static void readDependenciesContainer(JsonReader reader, Map<String, Mod
 		while (reader.hasNext()) {
 			final String modId = reader.nextName();
 			final List<String> matcherStringList = new ArrayList<>();
+			/* @Nullable */
```

Comment:
No need for this comment, you can just use the annotation or leave it out because `reason = null` implies it.

Reply:
When did the annotation start working in loader?

Reply:
Should work fine, Im fairly sure loader uses an up to date version of loom.

Reply:
Yeah, and it's also used in the changes to `ModDependencyImpl`.